service/history/queue/transfer_queue_processor.go

```diff
@@ -517,7 +528,6 @@ func newTransferQueueActiveProcessor(
 func newTransferQueueStandbyProcessor(
 	clusterName string,
 	shard shard.Context,
-	historyEngine engine.Engine,
```

Comment:
historyEngine parameter can also be removed for failover queue processor

Reply:
yes done